lectures/time_series_with_matrices.md

```diff
@@ -26,13 +28,12 @@ This lecture uses matrices to solve some linear difference equations.
 
 As a running example, we’ll study a **second-order linear difference
 equation** that was the key technical tool in Paul Samuelson’s 1939
-article {cite}`Samuelson1939` that introduced the **multiplier-accelerator** model.
+article {cite}`Samuelson1939` that introduced the *multiplier-accelerator model*.
 
 This model became the workhorse that powered early econometric versions of
 Keynesian macroeconomic models in the United States.
 
-You can read about the details of that model in [this](https://python.quantecon.org/samuelson.html)
-QuantEcon lecture.
+You can read about the details of that model in [Samuelson Multiplier-Accelerator](https://python.quantecon.org/samuelson.html).
```

Comment:
@SylviaZhaooo we should use `intersphinx` links for this. 

I will commit a change as an example

Reply:
resolved.